src/MediaInfoBundle/Resources/views/Download/windowsTable.html.twig

```diff
@@ -30,11 +35,16 @@
 </tr>
 
 <tr>
-    <th rowspan="3">32 bit</th>
-    <th><abbr title="Graphical User Interface">GUI</abbr></th>
+    <th rowspan="4">32 bit</th>
+    <th><abbr title="Graph visualization plugin for the GUI">Graph</abbr></th>
     <td><a href="{{ downloadLink }}">v21.09 Universal (32/64 bit)</a> <small> (<a href="//mediaarea.net/download/binary/mediainfo-gui/21.09/MediaInfo_GUI_21.09_Windows_i386_WithoutInstaller.7z">32 bit only without installer</a>)</small></td>
     <td></td>
 </tr>
+<tr>
+    <th>Graph vizualisation plugin for the GUI</th>
+    <td><a href="//mediaarea.net/download/binary/mediainfo-graph-plugin/21.09/MediaInfo_GraphPlugin_21.09_Windows.exe">v21.09</a></td>
```

Comment:
It was 2x. Need to change there too.
& 2x the comment part: `Manual download in case the automatic install from the GUI does not work.`

Reply:
I change a bit: `<abbr title="Graph visualization plugin for the GUI (3D audio only)">Graph</abbr>`